### PR TITLE
Transfer color docs text into mdx

### LINKED ISSUE #1302: docs: convert color page to have mdx headings
Will allow TOC to work and be linkable/shareable.

### PR DESCRIPTION
**Pull request by jaketrent**
Resolves #1302

### CODE REVIEW

File: docs/src/content/core/color.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Color
 route: /core/color
 ---
 
-import { ColorCategories } from '../../components/color/color-categories'
+import { Colors } from './color'
```
Comment (EdwardIrby) [inline docs/src/content/core/color.mdx:6]: I'm getting a warning on run yarn start:dev saying this import is not found. Color page is no longer rendering. I ran gatsby clean and the result is the same. 
Screen Shot 2020-10-07 at 9 08 37 AM

**Review by EdwardIrby — CHANGES_REQUESTED** [review]

File: docs/src/content/core/color.mdx (hunk 1 above)
Comment (jaketrent) [reply docs/src/content/core/color.mdx:6]: does a gatsby clean help for you?  I get this error too and that helps me.

Comment (jaketrent) [reply docs/src/content/core/color.mdx:6]: eh, scratch that.  Just tried again.  I still get the error.  Will address...

**Comment by jaketrent** [comment]
@EdwardIrby k, with latest tweak, gatsby clean && yarn start:dev works for me.

**Review by EdwardIrby — APPROVED** [review]

**Pull request merged**